resources/views/admin/settings/index.blade.php

```diff
@@ -10,7 +10,7 @@
         <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
           <h1 class="page-header">Settings</h1>
 
-          <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dolorem quam, voluptate, vero odio hic eum minima excepturi, accusantium delectus quasi veniam saepe quod vitae qui repudiandae quae aperiam rerum soluta.</p>
+          <p>Click below to create a new scholarship or edit the current one, or select another option from the sidebar.</p>
 
```

Comment:
@katiecrane ah ha! This was so annoying when I loaded the page but I didn't realize it was because of the Lorem ipsum! 💡